posthog/temporal/batch_exports/snowflake_batch_export.py

```diff
@@ -307,6 +331,11 @@ async def connect(self):
             self._connection = None
             await asyncio.to_thread(connection.close)
 
+    def ensure_snowflake_logger_level(self, level: str):
+        """Ensure the log level for logger used by inner `SnowflakeConnection`."""
+        logger = logging.getLogger("snowflake.connector")
+        logger.setLevel(level)
```

Comment:
Ok, structlog doesn't really have handlers in the python stdlib logging sense of the word. So, best we can do is set the level. I've made this DEBUG instead of the default "INFO".